docs/docs/cmd/entra/m365group/m365group-set.mdx

```diff
@@ -49,38 +61,56 @@ When updating group's owners and members, the command will add newly specified u
 
 When specifying the path to the logo image you can use both relative and absolute paths. Note, that ~ in the path, will not be resolved and will most likely result in an error.
 
+:::warning
```

Comment:
Maybe instead of a warning, let's use a `note` to make it less severe (we are validating this anyway).